java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -320,6 +321,18 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
     let sourceForContentProvider = new NetBeansSourceForContentProvider();
     context.subscriptions.push(vscode.workspace.registerTextDocumentContentProvider('sourceFor', sourceForContentProvider));
 
+    // initialize Run Configuration
+    initializeRunConfiguration().then(initialized => {
+		if (initialized) {
+			context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('java8+', runConfigurationProvider));
```

Comment:
what about `nativeimage` type executions ? They can also have cwd, env, args.

Reply:
Support for the Native Image launcher has not been considered for the initial release. Might be useful as well, please file a RFE.